### DISCUSSION TITLE
Logic behind the "Unified Telemetry Client Outbound Traffic"

### DISCUSSION THREAD

**Discussion by savchenko** [farag2/Sophia-Script-for-Windows#499] [Q&A] [answered]
In [Sophia.psm1](https://github.com/farag2/Sophia-Script-for-Windows/blob/1ec329d30528a325aee128ce025a7e92f9235232/src/Sophia_Script_for_Windows_10_LTSC_2019/Module/Sophia.psm1#LL960C1-L960C1):

```ps1
# Block connection for the Unified Telemetry Client Outbound Traffic
Get-NetFirewallRule -Group DiagTrack | Set-NetFirewallRule -Enabled False -Action Block
```

Which doesn't make sense as the disabled rule won't block the connection. Shouldn't it be:

```ps1
Get-NetFirewallRule -Group DiagTrack | Set-NetFirewallRule -Enabled True -Action Block
```

?

**Comment by farag2** [comment, marked as answer]
Hm, it seems you're right. :) Could you create a general PR? 
And hi. :cat: I didn't think you had wanted to dig into the code.

**Reply by savchenko** [reply]
Done: https://github.com/farag2/Sophia-Script-for-Windows/pull/500
